gradle.properties

```diff
@@ -0,0 +1 @@
+defaultScalaVersion=2.13.11
```

Comment:
Prefer to use https://stackoverflow.com/questions/12473537/naming-convention-for-properties-in-property-file